src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -32,16 +32,28 @@
  * @see <a href="https://gist.github.com/rherrmann/7447571">Starting point</a>
  */
 public class ConditionalIgnoreRule implements MethodRule {
+	private static <A extends Annotation> Stream<A> annotation(Annotatable source, Class<A> type) {
+		return Stream
+			.of(source.getAnnotation(type))
+			.filter(Objects::nonNull);
+	}
+
+	private static Stream<ConditionalIgnore> annotations(Annotatable source) {
+		return Stream.concat(
+			annotation(source, ConditionalIgnore.class),
+			annotation(source, ConditionalIgnoreList.class).map(ConditionalIgnoreList::value).flatMap(Stream::of)
+		);
+	}
+
 	@Override
 	public Statement apply(Statement base, FrameworkMethod method, Object target) {
-		ConditionalIgnore annotation = method.getAnnotation(ConditionalIgnore.class);
-		if (annotation != null) {
-			IgnoreCondition condition = createCondition(target, annotation);
-			if (condition.needSkip()) {
-				return new IgnoreStatement(condition);
-			}
-		}
-		return base;
+		return annotations(method)
```

Comment:
\[Checkstyle\] WARNING: 'method def' child has incorrect indentation level 16, expected level should be 4\.